src/content/learn/you-might-not-need-an-effect.md

```diff
@@ -1600,17 +1602,17 @@ button {
 
 </Solution>
 
-#### Submit a form without Effects {/*submit-a-form-without-effects*/}
+#### Efektler olmadan bir form gönderin {/*submit-a-form-without-effects*/}
 
-This `Form` component lets you send a message to a friend. When you submit the form, the `showForm` state variable is set to `false`. This triggers an Effect calling `sendMessage(message)`, which sends the message (you can see it in the console). After the message is sent, you see a "Thank you" dialog with an "Open chat" button that lets you get back to the form.
+Bu `Form` bileşeni bir arkadaşınıza mesaj göndermenize izin verir. Formu gönderdiğinizde, `showForm` state değişkeni `false` olarak değişir. Bu `sendMessage(message)` adında  mesaj gönderen bir Efekt tetikler (mesajı konsolda görebilirsiniz). Mesaj gönderildikten sonra, forma geri dönmenizi sağlayan "Open chat" butonu olan bir "Thank you" iletişim kutusunu görürsünüz.
 
-Your app's users are sending way too many messages. To make chatting a little bit more difficult, you've decided to show the "Thank you" dialog *first* rather than the form. Change the `showForm` state variable to initialize to `false` instead of `true`. As soon as you make that change, the console will show that an empty message was sent. Something in this logic is wrong!
+Uygulamanızın kullanıcıları çok fazla mesaj gönderiyor. Sohbet etmeyi biraz daha zorlaştırmak için, form yerine *önce* "Thank you" iletişim kutusunu göstermeye karar verdiniz. Bunun için `showForm` state değişkenini `true` yerine `false` ile başlatacak şekilde değiştirin. Bu değişikliği yaptığınız anda konsol boş bir mesajın gönderildiğini gösterecektir. Bu mantıkta bir şeyler yanlış!
 
-What's the root cause of this problem? And how can you fix it?
+Bu hatanın ana kaynağı nedir? Ve bunu nasıl düzeltebilirsiniz?
 
 <Hint>
 
-Should the message be sent _because_ the user saw the "Thank you" dialog? Or is it the other way around?
+Kullanıcının "Thank you" iletişim kutusunu görmesi _nedeniyle_ mesaj gönderilmeli mi? Yoksa tam tersi mi geçerlidir?
```

Comment:
Bu daha anlasilir olabilir:

`Kullanıcı "Thank you" mesajını _gördüğü için_ mi mesaj gönderilmeli? Yoksa tam tersi, mesajı gönderdiği için mi "Thank You" mesajını görmeli?`